proto/streamlit/proto/NewSession.proto

```diff
@@ -151,15 +155,15 @@ message FontFace {
 
 message Radii {
   // In pixels.
-  int32 baseWidgetRadius = 1;
-  int32 checkboxRadius = 2;
+  int32 base_widget_radius = 1;
+  int32 checkbox_radius = 2;
 }
 
 message FontSizes {
   // In pixels.
-  int32 tinyFontSize = 1;
-  int32 smallFontSize = 2;
-  int32 baseFontSize = 3;
+  int32 tiny_font_size = 1;
+  int32 small_font_size = 2;
+  int32 base_font_size = 3;
```

Comment:
Hm, looks like these proto fields being named in `camelCase` instead of `snake_case` must have slipped by code review. I think this change won't actually affect anything as I'm guessing these protos are only ever set in javascript environments, in which case the proto compiler will convert the field names to `camelCase` for us.

Would be worth confirming with @sfc-gh-tteixeira that these types are only constructed by JS code, though.

Reply:
Verified that this change didn't affect our the custom theme set via `frontend/hostframe.html`